daprdocs/content/en/developing-applications/building-blocks/service-invocation/howto-invoke-discover-services.md

```diff
@@ -199,11 +199,14 @@ namespace EventService
                System.Threading.Thread.Sleep(5000);
                Random random = new Random();
                int orderId = random.Next(1,1000);
```

Comment:
Nit: We should be consistent with using `var` like we have in the rest of the example.

Also, instead of `Thread.Sleep` you should use `await Task.Delay(5000)` in an async method.